mllib/src/main/scala/org/apache/spark/ml/feature/ElementwiseProduct.scala

```diff
@@ -33,21 +33,26 @@ import org.apache.spark.sql.types.DataType
  * multiplier.
  */
 @Experimental
-class ElementwiseProduct(override val uid: String)
+@Since("1.4.0")
+class ElementwiseProduct @Since("1.4.0") (@Since("1.4.0") override val uid: String)
   extends UnaryTransformer[Vector, Vector, ElementwiseProduct] with DefaultParamsWritable {
 
+  @Since("1.4.0")
   def this() = this(Identifiable.randomUID("elemProd"))
 
   /**
    * the vector to multiply with input vectors
    * @group param
    */
+  @Since("1.4.0")
```

Comment:
good point, and also yeah technically all unary transformers over `Vector` should probably be since `2.0.0`. Will update